src/services/admission/admission-router.test.ts

```diff
@@ -0,0 +1,86 @@
+import { beforeEach, describe, expect, it } from "@jest/globals";
+import Models from "../../database/models.js";
+import { DecisionStatus, UpdateEntries } from "./admission-formats.js";
+import { getAsAttendee, getAsStaff, getAsUser, putAsAttendee, putAsStaff, putAsUser } from "../../testTools.js";
+import { DecisionInfo } from "../../database/decision-db.js";
+
+beforeEach(async () => {
+    Models.initialize();
+});
+
+describe("GET /admission", () => {
+    it("gives forbidden error for user without elevated perms", async () => {
+        const responseUser = await getAsUser("/admission/").expect(403);
+        const responseAttendee = await getAsAttendee("/admission/").expect(403);
+        expect(JSON.parse(responseUser.text)).toHaveProperty("error", "InvalidToken");
+        expect(JSON.parse(responseAttendee.text)).toHaveProperty("error", "InvalidToken");
+    });
+
+    it("should return a list of applicants without email sent", async () => {
```

Comment:
Correct me if I'm wrong but there's no actual decisions that exist, so isn't this just testing if entries exists? Also, use expect(JSON.parse(response.text)).toBe(something) instead of iterating